src/foam/nanos/auth/ServiceProviderAwareDAO.js

```diff
@@ -115,7 +96,7 @@ foam.CLASS({
       }
       if ( SafetyUtil.isEmpty(spid) ||
            user != null &&
-           user.isAdmin() &&
+           auth.check(x, "default.to.theme.spid") &&
```

Comment:
```suggestion
           auth.check(x, "theme.read.*") &&
```
We don't have theme permissions, but if we had to add them in the future.

Reply:
I don't think keeping this similar to our DOPs would benefit here since it's a condition placed on whether the user should default to the spid of the theme applied to context. It doesn't share much relevance to reading themes.